### DISCUSSION TITLE
Run 2D and 3D_fullres training using separate slurm scripts at the same time

### DISCUSSION THREAD

**Discussion by jhogan7** [MIC-DKFZ/nnUNet#950] [Q&A] [answered]
I have access to multiple supercomputer GPUs over slurm. I know that multi-GPU training is not recommended, and am training all folds of 3D_full_res using a single GPU (same script). Would it be possible to submit another slurm script at the same time training the model with a 2D configuration using a separate GPU? 
The output will be written to two different folders; however, I am not sure if there will be any issues with reading files as the training data are the same.

Thanks!

**Comment by jhogan7** [comment]
In answer to my own question
![image](https://user-images.githubusercontent.com/54959376/156246042-d825f036-898b-4557-9123-322379ddd83a.png)

**Comment by FabianIsensee** [comment, marked as answer]
Just send 5 jobs, each for one of the folds? Heck you can even submit 15 jobs (5x 2d, 3dlowres, 3dfullres) at once :-)
